### ISSUE TITLE
image/icon 

### ISSUE THREAD

**Issue by edoz90** [jaagr/polybar#488]
Is it possibile to display an image on the bar?
i.e. the album art for spotify

**Comment by jaagr** [comment]
This is currently not supported.

**Comment by edoz90** [comment]
Hi,

Planning to support this feature?

**Comment by jaagr** [comment]
Yes, @jereksel is working on some features that will add support for this.

**Comment by HolyStephano** [comment]
Any status on this being available?

**Comment by patrick96** [comment]
@HolyStephano not much progress so far

**Comment by Zanark** [comment]
Any updates?

**Comment by patrick96** [comment]
See #1088. Probably still some time to go.

**Comment by karma-riuk** [comment]
Are there any news on the subject?
